scripts/k8s/getOffset.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python
+
+from kazoo.client import KazooClient
+import json
+import sys
+import time
+
+topic = sys.argv[1]
+
+zk = KazooClient(hosts='zookeeper:2181')
+zk.start()
+try:
+  resp = zk.get("/config/topics/%s" % topic)
```

Comment:
I guess it's safe to assume that kafka won't change that too often

Reply:
yeah, this wont change.   Kafka 0.11 and newer have an AdminApi for querying/setting topic config, but there are no clients for it yet.